TasteBuddiesServer/src/main/java/org/launchcode/TasteBuddiesServer/controller/EventController.java

```diff
@@ -72,12 +73,34 @@ public ResponseEntity<?> collectRestaurantData(@RequestBody EventDTO eventDTO)
         transcriptNB = gson.fromJson(getResponseNB.body(), TranscriptNB.class);
 
         List<String> place_ids = new ArrayList<>();
+        String pageToken = transcriptNB.getNext_page_token();
 
         for(int i=0; i < transcriptNB.getResults().size(); i++){
             String place_id = transcriptNB.getResults().get(i).getPlace_id();
             place_ids.add(place_id);
         }
 
+        while(pageToken != null){
+            Date currentDate = new Date();
+            Date futureDate = new Date();
+            futureDate.setTime(currentDate.getTime()+2000);
+            while(futureDate.getTime() > currentDate.getTime()){
```

Comment:
This is fine for the moment, but I found a better way to do this - using a ScheduledExecutorService: [https://stackoverflow.com/questions/24104313/how-do-i-make-a-delay-in-java](https://stackoverflow.com/questions/24104313/how-do-i-make-a-delay-in-java)